### ISSUE TITLE
Enhancement - Port consoles 

### ISSUE THREAD

**Issue by nunoguerreiro** [moloch--/RootTheBox#352]
Any thoughts when this can be integrated ?

https://github.com/moloch--/RootTheBox/issues/40 #40 
that will be awesome.

**Comment by jeffg2k** [comment]
Guess I'd just need to understand the use case and how it would work in game.  Like, why not just have them ssh to the box?

**Comment by nunoguerreiro** [comment]
@jeffg2k - yes that is one of the options.

Other, is to open a web terminal directly from the box (if there is an ip or domain associated for that box) directly and have a terminal emulation inside.

**Comment by jeffg2k** [comment]
So if I understand correctly, it's sort of a ssh proxy that uses a web interface.. I can see how that would be useful if you wanted to limit backend visibility and make it easy for the player. Not sure when I'd get to that feature, but I'll add it to the list.  I have a few others I want to get to first.

**Comment by nunoguerreiro** [comment]
> So if I understand correctly, it's sort of a ssh proxy that uses a web interface.. I can see how that would be useful if you wanted to limit backend visibility and make it easy for the player. Not sure when I'd get to that feature, but I'll add it to the list. I have a few others I want to get to first.

Yes. exactly the use case... see the box ip known addresses screenshot.
![Untitled](https://user-images.githubusercontent.com/1771943/75680076-3153e980-5caa-11ea-9e27-e7e40cac3b13.png)

**Comment by nunoguerreiro** [comment]
Hi @jeffg2k any updates ? Thanks

**Comment by jeffg2k** [comment]
Not yet, but you could create your own and add it as an option from the tools menu using the config option `tool_links`.  Then they could ssh from that web terminal to whatever internal address you want.
